### PR TITLE
Send email on low stock

### PR DESCRIPTION
**Pull request by Draakje10**
You need to use RubenDB for this pr

### CODE REVIEW

File: src/main/resources/templates/stocknotification.html
Comment (ronp2001) [inline src/main/resources/templates/stocknotification.html:1]: so many commented code sections that i stopped pointing them all out individually

**Review by ronp2001 — COMMENTED** [review]
this probably works fine, noticed a bunch of commented code also let me know what i need to change about the current db so we do not have to change over. (like do i only need to add an email and notification boolean to an account and does it work?)

File: src/main/resources/templates/stocknotification.html
Comment (Draakje10) [reply src/main/resources/templates/stocknotification.html:1]: Removed the comments and everything new in the database is the notification boolean and the email in accounts
afbeelding

**Comment by Draakje10** [comment]
> Approved, and again please let me know what to change about our current db to get this working for the rest of the group

d everything new in the database is the notification boolean and the email in accounts
295184253-b299373c-c823-42d1-8f6c-4573f3b54271

File: src/main/java/com/hsleiden/vdlelie/services/EmailService.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
+
+        mailSender.setUsername("draakje11111@gmail.com");
+        mailSender.setPassword("jkww pvwu suku pdoe ");
```
Comment (DaaromThomas) [inline src/main/java/com/hsleiden/vdlelie/services/EmailService.java:42]: Normaal password?

**Review by DaaromThomas — APPROVED** [review]

File: src/main/java/com/hsleiden/vdlelie/services/EmailService.java (hunk 1 above)
Comment (Draakje10) [reply src/main/java/com/hsleiden/vdlelie/services/EmailService.java:42]: random generated api password dus je kan niet zomaar mijn email in alleen beetje kut dat de repo open is maarja ik retract hem wel als wij klaar zijn met de opdracht

**Pull request merged**